syntax = "proto3";
package workflow;

import "parameter.proto";

message IntentOutput {
  Parameter parameter = 1;
}